conans/test/command/invalid_configuration_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import platform
+import os
+
+from conans.test.utils.tools import TestClient, TestServer
+from conans.model.ref import ConanFileReference, PackageReference
+from conans.paths import CONANFILE, CONANINFO
+from conans.model.info import ConanInfo
+from conans.test.utils.cpp_test_files import cpp_hello_conan_files
+from conans.paths import CONANFILE_TXT
+from conans.client.conf.detect import detected_os
+from conans.util.files import load, mkdir, rmdir
+
+
+class InvalidConfigurationTest(unittest.TestCase):
+
+    def setUp(self):
+        self.client = TestClient()
+        self.client.save({"conanfile.py": """
+from conans import ConanFile
+from conans.errors import ConanInvalidConfiguration
```

Comment:
Not sure, but maybe `from conans import ConanInvalidConfiguration` is better if we want to encourage its usage?